### ISSUE TITLE
Disconnecting USB causes IDE to use 100% CPU on Linux

### ISSUE THREAD

**Issue by nickgammon** [arduino/Arduino#2333]
On the Ubuntu version of the Arduino IDE  1.0.5, disconnecting (unplugging) the USB cable from the PC to the Arduino Uno causes CPU usage for the Java IDE runtime to jump to 100%. What is it doing?

Code:

```
top - 09:43:14 up 1 day, 23:54,  4 users,  load average: 0.71, 0.60, 0.60
Tasks: 246 total,   1 running, 244 sleeping,   0 stopped,   1 zombie
%Cpu(s):  8.8 us,  5.7 sy,  0.0 ni, 85.5 id,  0.1 wa,  0.0 hi,  0.0 si,  0.0 st
KiB Mem:  16376056 total, 16104688 used,   271368 free,   641840 buffers
KiB Swap: 31249404 total,        0 used, 31249404 free.  9019168 cached Mem

  PID USER      PR  NI    VIRT    RES    SHR S  %CPU %MEM     TIME+ COMMAND     
 2142 nick      20   0 6874812 163476  25408 S 100.1  1.0   0:27.44 java        
                                               ^^^^^
```

Steps to reproduce:
- Compile and upload Blink sketch
- Open Serial monitor
- Run the "top" command in a console window
- Unplug the USB cable from the Arduino
- Note that the java process CPU climbs from around 7% to 100%

**Comment by christakahashi** [comment]
Happens in windows arduino 1.6.12 too.  :(

**Comment by bperrybap** [comment]
This doesn't happen for me any more. Not sure what release fixed it, but the issue seems to have been fixed for me on linux mint a few releases ago.
Maybe it was fixed when the IDE started re-connecting to the serial port after uploading?

**Comment by grable0** [comment]
This happens for me using 1.8.5 on Windows 10, i figured out that its the Serial Monitor.
When its open and the cable is disconnected (the board doesnt even have to open the port) cpu usage jumps to around 15%, connect the board and it goes down again.
Probably stuck with some polling it does on the com port which no longer exists after the cable is unplugged...

**Comment by facchinm** [comment]
Should be solved by #8046, please reopen if the issue still occurs
